<ui version="4.0" >
 <class>FitWinBase</class>
 <widget class="QDialog" name="FitWinBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Fit settings</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QComboBox" name="FitType" >
       <item>
        <property name="text" >
         <string>S-curve fit (erfc)</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>TOT-calibration fit</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Linear fit</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Exponential</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>1-exp(-x)</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Double-exponential</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Constant</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Parabola</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Gaussian</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Simple S-curve fit (linear)</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>S-curve fit (Virzi, likelihood)</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>S-curve fit (Virzi, chi2)</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Cubic fit</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="Q3GroupBox" name="groupBox3" >
       <property name="title" >
        <string>Chi^2 settings</string>
       </property>
       <layout class="QVBoxLayout" >
        <item>
         <layout class="QVBoxLayout" >
          <item>
           <layout class="QHBoxLayout" >
            <property name="spacing" >
             <number>6</number>
            </property>
            <property name="margin" >
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="ChiLabel" >
              <property name="text" >
               <string>Chi^2 cut-off</string>
              </property>
              <property name="wordWrap" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="Spacer2" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QSpinBox" name="ChiCut" >
              <property name="minimumSize" >
               <size>
                <width>90</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximum" >
               <number>999999999</number>
              </property>
              <property name="value" >
               <number>200</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="divndfBox" >
            <property name="text" >
             <string>store chi^2/ndf (off: store raw chi^2)</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="Q3GroupBox" name="LimitGroup" >
       <property name="title" >
        <string>Limited fit range (min=max=0 -> no limit)</string>
       </property>
       <layout class="QHBoxLayout" >
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="TextLabel1_2_2" >
            <property name="text" >
             <string>Minimum</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="FitXmin" >
            <property name="maximum" >
             <number>1000000</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="TextLabel1_2_3" >
            <property name="text" >
             <string>Maximum</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="FitXmax" >
            <property name="maximum" >
             <number>1000000</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="Q3GroupBox" name="ConversionBox" >
       <property name="title" >
        <string>Conversion DAC->physical value</string>
       </property>
       <layout class="QHBoxLayout" >
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <widget class="QCheckBox" name="ChipCalBox" >
              <property name="text" >
               <string>calibrate per chip</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Q3Table" name="ChipCfac" >
              <property name="enabled" >
               <bool>true</bool>
              </property>
              <property name="minimumSize" >
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="frameShape" >
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="numRows" >
               <number>16</number>
              </property>
              <property name="numCols" >
               <number>4</number>
              </property>
              <row>
               <property name="text" >
                <string>1</string>
               </property>
              </row>
              <row>
               <property name="text" >
                <string>2</string>
               </property>
              </row>
              <row>
               <property name="text" >
                <string>3</string>
               </property>
              </row>
              <row>
               <property name="text" >
                <string>4</string>
               </property>
              </row>
              <row>
               <property name="text" >
                <string>5</string>
               </property>
              </row>
              <row>
               <property name="text" >
                <string>6</string>
               </property>
              </row>
              <row>
               <property name="text" >
                <string>7</string>
               </property>
              </row>
              <row>
               <property name="text" >
                <string>8</string>
               </property>
              </row>
              <row>
               <property name="text" >
                <string>9</string>
               </property>
              </row>
              <row>
               <property name="text" >
                <string>10</string>
               </property>
              </row>
              <row>
               <property name="text" >
                <string>11</string>
               </property>
              </row>
              <row>
               <property name="text" >
                <string>12</string>
               </property>
              </row>
              <row>
               <property name="text" >
                <string>13</string>
               </property>
              </row>
              <row>
               <property name="text" >
                <string>14</string>
               </property>
              </row>
              <row>
               <property name="text" >
                <string>15</string>
               </property>
              </row>
              <row>
               <property name="text" >
                <string>16</string>
               </property>
              </row>
              <column>
               <property name="text" >
                <string>1</string>
               </property>
              </column>
              <column>
               <property name="text" >
                <string>2</string>
               </property>
              </column>
              <column>
               <property name="text" >
                <string>3</string>
               </property>
              </column>
              <column>
               <property name="text" >
                <string>4</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <spacer name="spacer5" >
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>21</width>
                <height>71</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <widget class="QLineEdit" name="CfacEdit" >
              <property name="font" >
               <font/>
              </property>
              <property name="text" >
               <string>44.6</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="LoadButt" >
              <property name="enabled" >
               <bool>true</bool>
              </property>
              <property name="text" >
               <string>Load from file</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer4" >
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>20</width>
                <height>21</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QVBoxLayout" >
              <item>
               <widget class="QLabel" name="autoCalNote" >
                <property name="font" >
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text" >
                 <string>calibration is taken</string>
                </property>
                <property name="wordWrap" >
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="autoCalNote2" >
                <property name="font" >
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text" >
                 <string>from logfile</string>
                </property>
                <property name="wordWrap" >
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="spacer3" >
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>21</width>
                <height>81</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="OKButt" >
         <property name="text" >
          <string>Start Fit</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="Spacer1" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="CancButt" >
         <property name="text" >
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3Table</class>
   <extends>Q3Frame</extends>
   <header>q3table.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
